src/main/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtils.java

```diff
@@ -451,6 +451,19 @@ public static StorageAPIAvroReaderAndBigQueryStatistics executeQueryWithStorageA
     }
 
     public static boolean doRowsExistFor(String projectID, String datasetName, String tableName, String columnName, String value) {
```

Comment:
This isn't unique to this PR, but should we be worried about injection attacks here? It seems like we should be sanitizing the inputs and probably using whatever bigquery query builder exists instead of directly munging strings from the user into our queries.

Reply:
It's probably not clear from the way GitHub displays these diffs, but this PR is adding a second version of `doRowsExistFor` that makes its last parameter a `Long` instead of the `String`. So the changes here should be even safer than the baseline. 🙂 

The code in this file is often formatting not just column values but project / dataset / table. All of these are variable in GVS (even table names, thanks to "superpartitioning" to work around BQ partitioning constraints).

At least in the GVS context I'm not particularly concerned about injection here for a few reasons:

* The project and dataset must have been validated to even create the tables in the first place. 
* Table and column names are not specified from user input.
* Users would be running GVS via their pet service accounts and wouldn't have the ability to read or write anything that isn't theirs.
* These `doRowsExistFor` are only called to validate sample ids (Longs) and digests. So even if the VCF header had some "little Bobby Tables" in it we would be querying for the digest of that.

Happy to add some docs / warnings to clarify for potential future users for whom these assumptions may not apply.